packages/pyright-internal/src/analyzer/service.ts

```diff
@@ -298,7 +298,7 @@ export class AnalyzerService {
         this._backgroundAnalysisProgram.setConfigOptions(configOptions);
 
         this._executionRootUri = configOptions.projectRoot;
-        this._applyConfigOptions(host);
+        this.applyConfigOptions(host);
```

Comment:
this might be stomping a change by eric

Reply:
No this is the change I made. It needed to be protected so we can see when config options change.